librz/util/getopt.c

```diff
@@ -48,7 +48,10 @@ RZ_API int rz_getopt_next(RzGetopt *opt) {
 		 * if the user didn't specify '-' as an option,
 		 * assume it means -1.
 		 */
-		if (opt->opt == '-') {
+		if (opt->opt == (int)'P' && *place == 'm') {
```

Comment:
I'm not sure such a check makes sense here... The fact that `-Pm` *might* have a special meaning in the caller of this API should not mean that there is a special case here.